cli/command/configure/cmd.go

```diff
@@ -0,0 +1,142 @@
+package configure
+
+import (
+	"errors"
+	"fmt"
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/cli/command"
+	"github.com/saucelabs/saucectl/cli/credentials"
+	"github.com/spf13/cobra"
+	"github.com/tj/survey"
+	"os"
+	"regexp"
+)
+
+var (
+	configureUse     = "configure"
+	configureShort   = "Configure your Sauce Labs credentials"
+	configureLong    = `Persist locally your Sauce Labs credentials`
+	configureExample = "saucectl configure"
+	cliUsername      = ""
+	cliAccessKey     = ""
+)
+
+// Command creates the `new` command
+func Command(cli *command.SauceCtlCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     configureUse,
+		Short:   configureShort,
+		Long:    configureLong,
+		Example: configureExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			log.Info().Msg("Start Configure Command")
+			if err := Run(cmd, cli, args); err != nil {
+				log.Err(err).Msg("failed to execute configure command")
+				os.Exit(1)
+			}
+		},
+	}
+	cmd.Flags().StringVarP(&cliUsername, "username", "u", "", "username,, available on your saucelabs account")
+	cmd.Flags().StringVarP(&cliAccessKey, "accessKey", "a", "", "accessKey, available on your saucelabs account")
+	return cmd
+}
+
+// explainHowToObtainCredentials explains how to get credentials
+func explainHowToObtainCredentials() {
+	fmt.Println("\nDon't have an account ? Signup here https://saucelabs.com/sign-up !")
+	fmt.Printf("Already have an account ? Get your username and access key here: https://app.saucelabs.com/user-settings\n\n\n")
+}
+
+// interactiveConfiguration expect user to manually type-in its credentials
+func interactiveConfiguration() (*credentials.Credentials, error) {
+	explainHowToObtainCredentials()
+	creds := getDefaultCredentials()
+
+	qs := []*survey.Question{
+		{
+			Name: "username",
+			Prompt: &survey.Input{
+				Message: "SauceLabs username",
+				Default: creds.Username,
+			},
+			Validate: func(val interface{}) error {
+				str, ok := val.(string)
+				if !ok {
+					return errors.New("invalid input")
+				}
+				re := regexp.MustCompile(`^[a-zA-Z0-9_\-\.\+]{2,70}$`)
+				if !re.MatchString(str) {
+					return errors.New("invalid username format")
```

Comment:
@FriggaHel I feel like if we're going to validate the user name we should give more info in regards on the error message. `Invalid username format` doesn't tell the user the problem. What do you think on here, or any error to give the user the link again to their settings.

Reply:
I updated the message & provided the link to get username.